docs/modules/develop/pages/backports.adoc

```diff
@@ -2,37 +2,122 @@
 
 The process for making backports is the following:
 
-. Check in the git commit history what was the latest backports on the branch that you are working with. https://github.com/decidim/decidim/commits/release/0.27-stable[Example for v0.27]. Copy the number of the last PR backported.
-. Search in GitHub for the this PR in the list of merged PRs of the `type: fix`. https://github.com/decidim/decidim/pulls?page=1&q=is%3Apr+sort%3Aupdated-desc+label%3A%22type%3A+fix%22+is%3Amerged[Example URL].
-. Check the PRs merged just before and after this one. You need to find the last ones that were not backported already to start with the backport process. A fast way for checking this out is to open the PRs and see the mentions from the backports in the GitHub references.
-. Once you have the list of the PRs that you want to backport, you can start with the process by using the https://github.com/decidim/decidim-maintainers_toolbox[decidim-backporter script]. This script mostly handles the branch creation, cherrypicking, labeling and pushing of the fix. For using it you will need to have a GitHub Personal Access Token.
+. Reproduce the error against develop
+. Develop a fix for the issue, starting from the develop
+. Open the Pull Request, adding the following Red labels: `type: fix`, `release: 0.xx`, in our example using `0.32.0.dev`, we would attach `type: fix`, `release: v0.31`.
+. If the fix affects multiple versions, attach the corresponding release tag ( ex: `release: v0.30`)
+. Be patient and wait for a maintainer to review and merge
+. Once it is merged, an automated workflow will attempt to apply the patch
+. If you have an already merged Pull Request, you can add another `release: v0.xx` tag, and this will retrigger the backport workflow. So you do not need to decide when you open the PR which are the versions you want to backport to.
 
-This is an example on how this command is called:
+== Maintainers only
 
-```shell
-$ gem install UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG
-$ decidim-backporter --github-token=GITHUB_TOKEN --pull-request-id=10248 --version-number=0.27
-```
+in order to ease up the backport process, you will need to install the `decidim-maintainers_toolbox` gem
 
-And this is the current full documentation for this script:
+[source,bash]
+----
+gem install decidim-maintainers_toolbox
+----
 
-```shell
-$ decidim-backporter --help
+This will provide you with some commands that you can use for day to day activities:
+
+Before starting to use `decidim-maintainers_toolbox` gem, you will need to make sure that you have available the `gh` command, provided by `GitHub`. Follow the https://github.com/cli/cli[project's page] for installation steps that are most suited for your Operating System.
+
+=== decidim-backports-checker
+
+The `decidim-backports-checker` is an utility that you can use to investigate which are the Pull Requests that needs to be backported.
```

Comment:
As the output for this one is basically 100% the whole idea of the tool, I think it'd be nice to have a screenshot of the ouput (not a blocker - just a nice to have).

We already have the output below, but mind that this tool relies in color coding, and that's not something that can't be done with asciidoctor/antora (or I couldn't find it)